src/main/java/com/google/devtools/build/lib/analysis/config/FeatureSet.java

```diff
@@ -44,6 +45,11 @@ private static FeatureSet of(Set<String> on, Set<String> off) {
 
   /** Parses a {@link FeatureSet} instance from a list of strings. */
   public static FeatureSet parse(Iterable<String> features) {
+    for (String feature : features) {
+      Preconditions.checkArgument(
+          !feature.contains(","),
```

Comment:
Is there a better way to gracefully warn & exit from FeatureSet?
I can also turn this into a function / utility class